api/api/controllers/search_controller.py

```diff
@@ -401,8 +395,8 @@ def build_collection_query(
     if not include_sensitive_by_params:
         search_query["must_not"].append({"term": {"mature": True}})
 
-    if excluded_providers_query := get_excluded_providers_query():
-        search_query["must_not"].append(excluded_providers_query)
+    if filtered_providers_query := get_filtered_providers_query():
+        search_query["must"].append(filtered_providers_query)
```

Comment:
Same as above, with respect to must vs filter.

```suggestion
        search_query["filter"].append(filtered_providers_query)
```